python/scripts/rfi_receiver/rfi_receiver.py

```diff
@@ -486,7 +489,9 @@ def bad_input_listener(thread_id):
     RFIHeaderSize = app.config["chime_rfi_header_size"]
     mode = app.mode
     firstPacket = True
-    PacketSize = RFIHeaderSize + 4 * local_freq * num_elements
```

Comment:
@andrerenard I think this line was originally wrong, as the `fault_counter` being sent by `kotekan` in `rfiBadInputFinder.cpp` is a byte array not an integer array. So it should be:
`PacketSize = RFIHeaderSize + local_freq * num_elements`

What do you think?

Reply:
I agree.  This looks like it's incorrect in the original code.